src/url-loader/package-url-resolver.ts

```diff
@@ -91,6 +125,44 @@ export class PackageUrlResolver extends UrlResolver {
       from = this.packageUrl;
       to = fromOrTo;
     }
+    return this.relativeImpl(from, to);
+  }
+
+  private relativeImpl(from: ResolvedUrl, to: ResolvedUrl): FileRelativeUrl {
```

Comment:
why is this extracted? is it used elsewhere?

Reply:
I wanted to isolate the fromOrTo / maybeTo handling from the implementation, for clarity. With the destructuring one-liner it's much better, but now there's the issue that `to` is written to but `from` is not, so `const` is an error and `let` is a tslint warning.